app/services/deploy_status_checker.rb

```diff
@@ -5,6 +5,21 @@ class DeployStatusChecker
   Environment = Struct.new(:env, :statuses)
 
   DEPLOYS = [
+    Deploy.new('identity-idp', 'prod', 'https://idp.prod.login.gov'),
```

Comment:
It should, but that name change hasn't been merged/deployed yet, so I figured it made sense to deploy the environment we currently have?